content/python/virtual_guest.py.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Working with Virtual Guest"
```

Comment:
I actually created a "Working with Virtual Guests" merged example already. https://github.com/softlayer/githubio_source/pull/367/files

Change the title to "More Virtual Guest Examples"

Reply:
Done.